docs/guides/software-lifecycle/security/github-security/README.md

```diff
@@ -0,0 +1,101 @@
+       1. Create a `.github/dependabot.yml` file in your repository.
+       2. Specify the package-ecosystem, directory, and schedule for the updates. For example:
+          ```yml
+          version: 2
+          updates:
+            - package-ecosystem: "npm"
+              directory: "/"
+              schedule:
+                interval: "daily"
+          ```
+   - To view Dependabot alerts and version updates:
+     - Head back to the main page of your repository.
+     - Click on the `Security` tab. Here, you can select `Dependabot alerts` to view security alerts, and you can see version updates in the `Pull requests` tab labeled with "Dependabot".
+
+
+2. **Enable Code Scanning:**
+   - In the `Code security and analysis` menu from the `Settings` tab, click the "Set Up" or enable the following workflows:
+     - _CodeQL Analysis workflow:_ a free tool provided by GitHub that scans your code for vulnerabilities across a variety of languages. Simply choose a CodeQL Analysis template (default is acceptable) and follow the instructions.
+   - To view Code scanning alerts:
+     - Return to the repository main page.
+     - Click on the `Security` tab and select `Code scanning alerts`.
+
+3. **Enable Secret Scanning:**
+   - In the `Code security and analysis` menu from the `Settings` tab:
+     - Click on the `Secret scanning` enable button.
+     - We recommend enabling "Push protection" for blocking commits containing secrets
+   - To view Secret scanning alerts:
+     - Navigate to the repository main page.
+     - Click on the `Security` tab and select `Secret scanning alerts`.
+
+## Frequently Asked Questions (FAQ)
+
+- **Q: Can these security features be used outside of GitHub?**
+
+  A: This guide specifically focuses on GitHub’s ecosystem. While some tools might have external equivalents, the integrations and configurations here are GitHub-specific.
+
+- **Q: If I receive security alerts, what should I do and how soon should I act?**
+
+  A: When you receive a security alert, it indicates a potential vulnerability in your repository. First, review the details of the alert to understand the severity and the affected component. Address critical vulnerabilities immediately, as they can pose a significant risk to your project. For less severe alerts, plan to address them in a timely manner. Always keep in mind that the sooner you act on security alerts, the better you can protect your code and users from potential threats.
+  
+## Credits
+
+**Authorship:**
+- John Engelke [@jpl-jengelke](http://github.com/jpl-jengelke/)
+- Rishi Verma [@riverma](http://github.com/riverma/)
+
+**Acknowledgements:**
+- GitHub for providing the security features and related documentation. See [GitHub’s Security Features](https://docs.github.com/en/code-security) to access an overview of the suite of security features GitHub provides for repositories.
+
```

Comment:
I would add: 
- The [OWASP DevSecOps Guideline](https://owasp.org/www-project-devsecops-guideline/latest/) for providing a Shift Left strategy to secure all phases of development.

Why? What we are really doing here is implementing DevSecOps. (See https://www.infoworld.com/article/3681097/what-is-devsecops-securing-devops-pipelines.html .)

Reply:
Thanks @jpl-jengelke! Added. By the way, in the future if you have additions / modification suggestions, could you please use the code suggestion feature?

![Screenshot 2023-10-18 at 3 17 45 PM](https://github.com/NASA-AMMOS/slim/assets/3129134/7e65d713-417d-4499-8960-c42f7e399f4f)

Reply:
Sure, will do next time. In the past I have been reticent to do that since some devs prefer comments rather than code changes.

Reply:
That makes sense. Yeah - we're all friends here :)